lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,58 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_nested_associations
+
         def serializable_hash(options = nil)
           options ||= {}
+
+          self._nested_associations = ActiveModel::Serializer::Utils
```

Comment:
Following a discussion with @bf4, let's not support this option at `serializable_hash` level, but only in the constructor (i.e. at adapter-level).